test/functional/change_stream.test.js

```diff
@@ -2399,7 +2401,7 @@ describe('Change Streams', function() {
             expect(token).to.deep.equal(resumeAfter);
           });
       });
-      it('must be empty if neither the startAfter nor resumeAfter options were specified', function() {
+      it('must be empty if resumeAfter options was not specified', function() {
```

Comment:
what's the purpose of this rename? It looks like the test hasn't changed at all

Reply:
The language of the prose tests changed in the spec, so I updated the language of the corresponding tests to match. Since this test doesn't specifically test the `startAfter` feature there wasn't any code to update.